tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -347,10 +348,10 @@ def test_tqdm_progress_bar_value_on_colab(tmpdir):
         [2, 3, 1, [1, 2, 3, 4, 5], [1, 2, 3]],
         [0, 0, 3, None, None],
         [1, 0, 3, [1], None],
-        [1, 1, 3, [1, 2], [1]],
```

Comment:
Can you explain why this changed?

Reply:
yes, so since `train batches = 1` and `val batches = 1` and refresh rate is 2, so ideally main progress bar should only update when total progress is 2 since it contains train+val.

Reply:
I disagree here. The refresh rate is meant to reduce step-level updates but I don't think it should apply to epoch-level updates.

With this change, metrics logged on_epoch do not appear in the training progress and I think that's counterintuitive for the user, even if it's a stricter view of the refresh rate concept.